tls-client/main.cpp

```diff
@@ -401,12 +412,11 @@ class HelloHTTPS {
     void onError(TCPSocket *s, int error) {
         printf("MBED: Socket Error: %d\r\n", error);
         s->close();
-        _error = true;
     }
 
 protected:
     TCPSocket* _tcpsocket;
-
+    
```

Comment:
There are whitespace characters on this empty line, could you please remove them?